### ISSUE TITLE
Replace markedj with Atlassian's commonmark library?

### ISSUE THREAD

**Issue by takezoe** [gitbucket/gitbucket#1307]
https://github.com/atlassian/commonmark-java

**Comment by McFoggy** [comment]
Yep why not ; I like the idea of commonmark being both normalized & open to extensions.

Would be cool to have also the possibility to activate new extensions:
- by configuration in `gitbucket.conf` + classpath modification (to integrate the extension)
- or via a plugin & a plugin hook

It could then less us integrate without any hack inside gitbucket some markup addons.

Does this new one remember you something ? ;-) 
- https://github.com/McFoggy/commonmark-ext-notifications
